code/__DEFINES/dcs/signals/signals_mob/signals_mob_spawner.dm

```diff
@@ -4,3 +4,6 @@
 
 /// called when a ghost clicks a spawner role: (mob/living)
 #define COMSIG_GHOSTROLE_SPAWNED "ghostrole_spawned"
+
+/// Whenever a ghostpoll component concludes. Returns a ghost
+#define COMSIG_GHOSTPOLL_CONCLUDED "ghostpoll_concluded"
```

Comment:
This shouldn't be a signal, it should just be a callback invoked when the job is done